src/components/Calculator.js

```diff
@@ -1,99 +1,90 @@
 /* eslint-disable react/prefer-stateless-function */
-import React from 'react';
+import React, { useState } from 'react';
 import '../App.css';
 import calculate from '../logic/calculate';
 import Button from './Button';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-    };
-    this.onClickHandler = this.onClickHandler.bind(this);
-  }
+function Calculator() {
+  const [state, setState] = useState({ total: null, next: null });
 
-  onClickHandler(e) {
-    this.setState((state) => calculate(state, e.target.innerHTML));
-  }
+  const onClickHandler = (e) => {
+    setState((state) => calculate(state, e.target.innerHTML));
+  };
 
-  render() {
```

Comment:
- [ ] You've done a pretty awesome job so far. However, when I double-clicked an operator, I got this browser error `TypeError: one.times is not a function` . Please, let's create an error-handling exception for this scenario.

[OPTIONAL]

- [ ] Always favor the use of ES6 syntax, in this case, the arrow function instead of the function keyword.